inc/azure_c_util/containing_record.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef CONTAINING_RECORD_H
+#define CONTAINING_RECORD_H
+
```

Comment:
Nice!

Reply:
Thanks!

---
In reply to: [385375187](https://github.com/Azure/azure-c-util/pull/16#discussion_r385375187) [](ancestors = 385375187)